mailer.rst

```diff
@@ -0,0 +1,648 @@
+Sending Emails with Mailer
+==========================
+
+.. versionadded:: 4.3
+    The Mailer component was added in Symfony 4.3. Previously, Swift Mailer
+    was recommended. You can still :doc:`read its documentation</email/swiftmailer>`.
+
+Installation
+------------
+
+Symfony's Mailer component is a *powerful* way to send email - complete with
```

Comment:
Here, you're talking about the Mailer AND the MIME component, not just the Mailer component. The difference is huge and important. Mailer is *only* about sending emails. MIME is about creating emails. Having one document to explain how to create and send emails is ok, but I think we need to keep "deep" docs in the component docs themselves.